api/mapping/services.py

```diff
@@ -174,26 +179,97 @@ def process_scan_report(scan_report_id):
                 frequency=frequency,
             )
 
+            DataDictionary.objects.create(
+                source_value=ScanReportValue.objects.latest("id"),
+                definition_fixed=False,
+            )
 
-def import_data_dictionary(filepath):
-    filepath = filepath
 
-    with open(filepath, 'rt') as f:
-        reader = csv.reader(f)
-        next(reader)  # Skip header row
+def build_usagi_index():
+    # Use 'build' to create index for the first time.
+    # Index stored in data/usagi/mainIndex
+    p = subprocess.Popen(
+        "java -jar Usagi.jar build usagi_build_index.properties",
```

Comment:
note to self: check Usagi.jar can be found

Reply:
Usagi.jar won't be present in this PR....it'll come a bit later on. At the moment, we can't run Usagi due to Docker performance issues.